server.js

```diff
@@ -1652,6 +1652,9 @@ cache(function(data, match, sendBadge, request) {
       } else if (status === 'uptodate') {
         badgeData.colorscheme = 'brightgreen';
         status = 'up-to-date';
+      } else if (status === 'none') {
+        badgeData.colorscheme = 'brightgreen';
+        status = 'none';
```

Comment:
There’s not really any need to set the status there, since the API already returned 'none' as the status, but otherwise that’s fine :+1: